src/main/java/ratpack/zipkin/Zipkin.java

```diff
@@ -0,0 +1,11 @@
+package ratpack.zipkin;
+
+import com.google.inject.BindingAnnotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@BindingAnnotation @Target({ElementType.PARAMETER}) @Retention(RetentionPolicy.RUNTIME)
```

Comment:
Could we also allow field injection? Personally I'm not a fan of field injection but it might be a convenient thing for users.